seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -529,4 +529,7 @@ avalanche_c,uniswap,3,2023-11-11,0x31e1a422f13f742ba7ad4c5b2b33bf2cc6c05777cfff4
 avalanche_c,uniswap,3,2023-11-11,0x41b6ec0e3ba96f6c72bab7954353152ba44904bd61232f91e824c9a74495b1e4,2,0x9702230a8ea53601f5cd2dc00fdbc13d4df4a8c7,600.415091,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,41.78982883
 optimism,mummy_finance,1,2023-06-20,0xcb520a60a0eae0a5a7b9e5658345151a370ef9c236072db32d9b3212ee24a496,36,0x4200000000000000000000000000000000000042,793.6112352289659,0x7f5c764cbc14f9669b88837ca1490cca17c31607,1048.751532
 optimism,mummy_finance,1,2023-06-30,0x671265ebe5dd61134633adf575ba80faec6be090271351ad79fed7c0f5468d50,65,0x4200000000000000000000000000000000000006,0.37616451992981625,0x4200000000000000000000000000000000000042,525.2047826760436
-optimism,mummy_finance,1,2023-06-09,0xd7cadb14f39a79bf0ca3ea0ebf9aa85caa1465da154010294ee502942a8de5d4,21,0x4200000000000000000000000000000000000006,1.530834771081591,0x4200000000000000000000000000000000000042,2097.160814236567
\ No newline at end of file
+optimism,mummy_finance,1,2023-06-09,0xd7cadb14f39a79bf0ca3ea0ebf9aa85caa1465da154010294ee502942a8de5d4,21,0x4200000000000000000000000000000000000006,1.530834771081591,0x4200000000000000000000000000000000000042,2097.160814236567
+optimism,firebird_finance,1,2023-06-21,0xee6c8e576530a282dd8b6b0150690d2efc9bafcf535b994a9efdd95c68d6aaf0,31,0x7f5c764cbc14f9669b88837ca1490cca17c31607,28.088612,0x47536f17f4ff30e64a96a7555826b8f9e66ec468,18.109264993439986
+optimism,firebird_finance,1,2022-09-12,0x4b229101a53eda64d7205e8c67da5b69ac57b7030d7420777131dcebd1b6451c,18,0x4200000000000000000000000000000000000006,0.01913919808305519,0x7f5c764cbc14f9669b88837ca1490cca17c31607,33.450938
+optimism,firebird_finance,1,2022-09-27,0x77266a3fb99476be6859436eb0000ec1d0a0a78400149195317a65e2a70a5577,11,0x7f5c764cbc14f9669b88837ca1490cca17c31607,400,0x4200000000000000000000000000000000000042,413.41739538888197
```

Comment:
Also you can move those seed records into this file:
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/aggregator/dex_aggregator_seed.csv